test/ufuzz.js

```diff
@@ -348,7 +348,7 @@ function createSwitchParts(recurmax, n, canThrow, canBreak, canContinue, cannotR
     var s = '';
     while (n-- > 0) {
         hadDefault = n > 0; // disables weird `default` clauses until handling stabilizes
-        if (hadDefault || rng(4) > 0) {
+        if (hadDefault || rng(10) > 0) {
```

Comment:
In the bugs it kept showing up first. That's mainly why I lowered it in hopes of seeing it in other positions. I'll put it to 20%.